lib/mayaUsd/fileio/utils/meshReadUtils.cpp

```diff
@@ -608,7 +624,25 @@ UsdMayaMeshReadUtils::assignPrimvarsToMesh(const UsdGeomMesh& mesh,
 
     // GETTING PRIMVARS
     const std::vector<UsdGeomPrimvar> primvars = mesh.GetPrimvars();
-    TF_FOR_ALL(iter, primvars) 
+
+    // Maya always has a map1 UV set. We need to find out if there is any stream in the file that
+    // will use that slot. If not, the first texcoord stream to load will replace the default map1
+    // stream.
+    bool hasDefaultUVSet = false;
+    TF_FOR_ALL(iter, primvars)
```

Comment:
We've been slowly yanking out usages of `TF_FOR_ALL` as we come across them, so these might be good ones to replace with regular range-based for loops:

```C++
for (const UsdGeomPrimvar& primvar : primvars) {
```